pkg/deploy/os_specific_windows.go

```diff
@@ -0,0 +1,21 @@
+package deploy
+
+import (
+	"strings"
+)
+
+func convertOsFileName(fileName string) string {
+	return strings.ReplaceAll(fileName, "_windows", "")
+}
+
+func osSpecificSkipFiles() map[string]bool {
+	return map[string]bool{
+		"ccm.yaml":             false, // i should figure out what this is
+		"coredns.yaml":         true,
+		"coredns_windows.yaml": false,
```

Comment:
Should we just do something like how go does os specific .go files?  So foo.yaml is all OSes, foo_linux.yaml and foo_windows.yaml is OS specific.  This way we can just manage things with the file names and not have to update this map.

Reply:
Made this change